src/core/sys/posix/locale.d

```diff
@@ -68,7 +76,7 @@ char*    setlocale(int category, const char* locale);
 /// Set the per-thread locale
 locale_t uselocale (locale_t locale);
 
-version (OSXBSDLocale)
+version (BSDLocale)
```

Comment:
```suggestion
version (DarwinBSDLocale)
```

To emphasize that this version still includes OSX & friends.

Reply:
The Posix layer in Darwin is from BSD. But ok.

Reply:
Done.